src/main/java/com/grafana/opentelemetry/OpenTelemetryConfig.java

```diff
@@ -41,20 +43,49 @@
 @PropertySource(value = {"classpath:grafana-otel-starter.properties"})
 public class OpenTelemetryConfig {
 
-  public static final String DISTRIBUTION_NAME = "telemetry.distro.name";
-  public static final String DISTRIBUTION_VERSION = "telemetry.distro.version";
+  public static final String PROTOCOL = "http/protobuf";
 
   private static final Logger logger = LoggerFactory.getLogger(OpenTelemetryConfig.class);
 
   public static final String OTLP_HEADERS = "otel.exporter.otlp.headers";
 
+  private static final Set<String> EXCLUDED_ATTRIBUTES =
+      Set.of(
+          ResourceAttributes.TELEMETRY_SDK_NAME.getKey(),
+          ResourceAttributes.TELEMETRY_SDK_LANGUAGE.getKey(),
+          ResourceAttributes.TELEMETRY_SDK_VERSION.getKey());
+
+  @Bean
+  public Resource getResource(AutoConfiguredOpenTelemetrySdk sdk) {
+    Method getResource = findMethod(AutoConfiguredOpenTelemetrySdk.class, "getResource");
```

Comment:
Can the OTel SDK make this public somehow?

Reply:
This has actually become inaccessible recently.

Eventually, this will be superceded by https://github.com/micrometer-metrics/micrometer/issues/4501